life_expectancy/cleaning.py

```diff
@@ -0,0 +1,136 @@
+"""Module for cleaning life expectancy data."""
+import argparse
+import pandas as pd
+
+def main(region: str) -> None:
+    """Main function."""
```

Comment:
Why didn't you put the region as Args in this docstring?

Reply:
yes indeed! missed it